trantor/tests/CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ set_property(TARGET ${targets_list} PROPERTY CXX_STANDARD 14)
 set_property(TARGET ${targets_list} PROPERTY CXX_STANDARD_REQUIRED ON)
 set_property(TARGET ${targets_list} PROPERTY CXX_EXTENSIONS OFF)
 
+find_package(Threads)
+
 foreach(T ${targets_list})
-  target_link_libraries(${T} PRIVATE trantor)
+  target_link_libraries(${T} PRIVATE trantor Threads::Threads)
```

Comment:
There are some lines in the main CMakeLists:
```
if(WIN32)
  target_link_libraries(${PROJECT_NAME} PRIVATE ws2_32 Rpcrt4)
  if(OpenSSL_FOUND)
    target_link_libraries(${PROJECT_NAME} PRIVATE Crypt32 Secur32)
  endif(OpenSSL_FOUND)
else(WIN32)
  target_link_libraries(${PROJECT_NAME} PRIVATE pthread)
endif(WIN32)
``` 
should we put this to private libraries of trantor?

Reply:
@BertrandD How do you think about this?

Reply:
Uh, I forgot to reply. Sorry.

You're right, I'll add Threads::Threads as private dep of trantor instead of pthread